src/network/protocol_handler.cpp

```diff
@@ -27,6 +27,14 @@ namespace network {
   //  bool ManageStartupPacket();
   void ProtocolHandler::SendInitialResponse() {}
 
+  bool ProtocolHandler::ProcessInitialPacket(UNUSED_ATTRIBUTE InputPacket* pkt,
```

Comment:
Why is this concrete and not pure virtual?

Reply:
@tli2 This may be because we are not quite clear about how to handle the initial packet of different protocols except wire protocol. I think we can make it virtual. Will double check it with @ChTimTsubasa